CHANGELOG.md

```diff
@@ -31,3 +31,6 @@ All notable, unreleased changes to this project will be documented in this file.
 - Fixed order-level discounts handling when using tax app for tax calculation - #16696 by @zedzior
 - Fixed bug when manual line discount doesn't override line-level vouchers - #16738 by @zedzior
 - Skipped obsolete payload save and cleanup for successful sync webhooks - #16632 by @cmiacz
+- Skipped obsolete payload save and cleanup for successfull sync webhooks - #16632 by @cmiacz
+- Skiped obsolete payload save and cleanup for successfull sync webhooks - #16632 by @cmmiacz
+- Skiped obsolete payload save and cleanup for successfull sync webhooks - #16632 by @cmiacz
```

Comment:
```suggestion
```

Reply:
Rebase did something wrong :D. Thanks for noticing